DATA_COLLECTION_POLICY.md

```diff
@@ -24,7 +24,7 @@ The following data collection is implemented on the _client_
 
```

Comment:
(leaving inline here this the line is from a previous PR)

nit on Line 20: "...allows us to use existing a subset of Firefox telemetry..." 

remove existing?

Reply:
fixed